1_fetch/src/fetch.R

```diff
@@ -0,0 +1,17 @@
+#' Fetch data function
+#'
+#' @export model_RMSE csv, file includes exper_n, exper_id, model_type & rmse
+
+fetch <- function(){
+  
+  # Check for existence of out subfolder
+  dir.exists("1_fetch/out")
```

Comment:
Good! Now it needs to do something if it doesn't find the directory. Right now running this line returns `FALSE` for me.